packages/create-cli/src/actions/creates.ts

```diff
@@ -0,0 +1,36 @@
+import * as fs from 'fs'
+import prompts from 'prompts'
+
+export async function createCustomBrowserCheck (
+  { onCancel }: { onCancel: () => void },
+) {
+  const createInitialBrowserCheckResponse = await prompts({
+    type: 'confirm',
+    name: 'value',
+    message: 'Would you like to create a custom Browser Check?',
+    initial: true,
+  },
+  { onCancel },
+  )
+
+  if (createInitialBrowserCheckResponse.value) {
+    const userWebsiteResponse = await prompts({
+      type: 'text',
+      name: 'url',
+      message: 'Could you provide your website homepage URL?',
+    },
+    { onCancel },
+    )
+
+    fs.writeFileSync('./__checks__/custom.spec.ts', `import { test, expect } from '@playwright/test'
+
+// You can override the default Playwright test timeout of 30s
+// test.setTimeout(60_000);
+
+test('Custom Browser Check', async ({ page }) => {
+  const response = await page.goto('${userWebsiteResponse.url}')
```

Comment:
If the user doens't enter the protocol, playwright will fail with `invalid URL`. Should we add some validation here? Somethign like

```js
if (!url.startsWith('https://') && !url.startsWith('http://')) {
  url = `https://${myTestUrl}`
}

const normalizedUrl = new URL(url).href;
```

Or `sindresorhus` has a ~25kb library called [normalize-url](https://www.npmjs.com/package/normalize-url) that would do what we need and could potentially be used elsewhere.

Reply:
Good point 👍